ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/StateTreeManager.java

```diff
@@ -59,8 +59,11 @@ public void onChainStarted(ChainStartEvent event) {
     // TODO: Startup Logic: Initial State Data
     List<Bytes32> merkle_roots = new ArrayList<Bytes32>();
     merkle_roots.add(Bytes32.ZERO);
+    List<Bytes48> proof_of_posession = new ArrayList<Bytes48>();
+    proof_of_posession.add(Bytes48.random());
+    proof_of_posession.add(Bytes48.random());
```

Comment:
Could we have a `Signature` type to abstract away the fact that signatures are represented as a 2 element list of Bytes48? It's a little confusing and possibly error-prone. Not for this PR, but a general observation.

Reply:
I think that is a great suggestion.  Definitely worth making an issue for this

Reply:
I opened an issue based on your suggestion:

https://github.com/PegaSysEng/artemis/issues/290